app/models/event.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Event < ApplicationRecord
+  belongs_to :creator, class_name: 'User', foreign_key: 'creator_id'
+  has_many :attendances, foreign_key: 'attended_event_id'
+  has_many :attendees, through: :attendances, foreign_key: 'attendee_id'
+
+  scope :past, -> { where('date < ?', DateTime.now) }
```

Comment:
Same for event creation, we need validations to the presence of all event attributes.

Reply:
Fixed by add validation to Event model.